src/components/Quote.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import '../style/Quote.css';
+
+export default function Quote() {
+  const [quote, setQuote] = useState('loading quote');
+  const [loading, setLoading] = useState('loading');
+  const fetchQuote = async () => {
+    try {
+      const response = await fetch('https://api.api-ninjas.com/v1/quotes?category=happiness', {
+        headers: { 'X-Api-Key': 'nezpEokeOVTwP1C5NwtGEQ==h62LjJXfJNexoQfy' },
+      });
+      const data = await response.json();
+      setQuote(data[0].quote);
+    } catch (error) {
+      setQuote(error);
+    }
```

Comment:
- [ ]  Good job for catching the error in your code, but the suggested and recommended way is to also have an error state. This means using `useState` to create an error state and then handling the error using that state.